### PR TITLE
Export function to validate a relational schema exists to allow extensions to leverage it

### PR DESCRIPTION
**Pull request by rafaelbey**
## Summary

While building extensions that leverage relational stores, there are cases when we need to validate a given schema exists.  This means duplication logic that already exists.  Exporting the function ensures consistency while doing this operation and eliminates duplication. 

## How did you test this change?

- [ ] Test(s) added
- [ ] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_DatabaseBuilderHelper.ts [hunk 1]
```diff
@@ -161,6 +161,15 @@ const schemaExists = (database: Database, _schema: string): boolean =>
   DEFAULT_DATABASE_SCHEMA_NAME === _schema ||
   _schemaExists(database, _schema, new Set<Database>());
 
+export const V1_assertSchemaExists = (
```
Comment (akphi) [inline packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_DatabaseBuilderHelper.ts:164]: To be consistent, let's rename this like the rest

```suggestion
export const V1_findSchema = (
```

File: packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_DatabaseBuilderHelper.ts [hunk 2]
```diff
@@ -193,10 +202,7 @@ export const V1_getRelation = (
   schemaName: string,
   relationName: string,
 ): Relation => {
-  assertTrue(
-    schemaExists(db, schemaName),
-    `Can't find schema '${schemaName}' in database '${db}'`,
-  );
+  V1_assertSchemaExists(db, schemaName);
```
Comment (akphi) [inline packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/to/helpers/V1_DatabaseBuilderHelper.ts:205]:
```suggestion
  V1_findSchema(db, schemaName);
```

File: .changeset/lucky-zoos-pay.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': minor
+---
+
+Export function to validate a relational schema exists on a given database to allow extensions to leverage it.
```
Comment (akphi) [inline .changeset/lucky-zoos-pay.md:5]:
```suggestion
Expose builder utility function `V1_findSchema()` to find a schema on a given database (including `included` databases).
```

**Review by akphi — DISMISSED** [review]

**Comment by akphi** [comment]
@rafaelbey tried to directly fixed the commit, but it seems I failed some formatting, could you pull the branch and run `yarn fix:format` please? thanks!

**Review by akphi — APPROVED** [review]

**Comment by akphi** [comment]
> @rafaelbey I tried to directly fix the commit, but it seems I failed some formatting 😢 , could you pull the branch and run `yarn fix:format` please? thanks!

Nevermind, I have merged this and push a fix for the formatting.

**Comment by rafaelbey** [comment]
Thanks @akphi

**Pull request merged**